doc/ref_cert/README.md

```diff
@@ -1,9 +1,136 @@
+## Test case integration requirements
+
+To reach all goals (verification, validation, compliance and conformance)
+expected by CNTT, all test cases must be delivered as
+[Docker containers](https://www.docker.com/) and meet the requirements to
+simplify the CI toolchain setups:
+- the common test case execution
+- the unified way to manage all the interactions with the CI/CD components and
+  with third-parties (e.g. dump all test case logs and results for
+  conformance)
+
+For their part, the Docker containers simply enforce that the test cases are
+delivered with all runtime dependencies. This prevents lots of manual
+operations when configuring the servers running the test cases and prevents
+conflicts between the test cases due to any dependencies.
+
+It's worth mentioning that current
+[test cases selected by CNTT](./chapter03.md)
+already leverages [Xtesting](https://xtesting.readthedocs.io/en/latest/)
+which is a simple framework to assemble sparse test cases and to accelerate the
+adoption of CI/CD best practices. By managing all the interactions with the
+CI/CD components (test scheduler, test results database, artifact repository),
+it allows the developer to work only on the test suites without diving into
+CI/CD integration. Even more, it brings the capability to run heterogeneous
+test cases in the same CI toolchains thanks to a few,
+[quickly achievable](https://www.sdxcentral.com/articles/news/opnfvs-6th-release-brings-testing-capabilities-that-orange-is-already-using/2018/05/),
+constraints.
 
+The Docker containers proposed by the test projects must also embed
+[the Xtesting Python package](https://pypi.org/project/xtesting/) and
+[the related test case execution description files](https://git.opnfv.org/functest-xtesting/tree/docker/testcases.yaml)
+as required by Xtesting.
+
+Here are the issues tracking the updates of the existing OPNFV test
+projects:
+- Bottlenecks: https://github.com/cntt-n/CNTT/issues/510
+- NFVBench: https://github.com/cntt-n/CNTT/issues/865
+- StorPerf: https://github.com/cntt-n/CNTT/issues/673
+- VSPERF: https://github.com/cntt-n/CNTT/issues/511
+- YardStick: https://github.com/cntt-n/CNTT/issues/509
+
+<a name="testing-cookbooks"></a>
+## Testing Cookbooks
+
+[Xtesting CI](https://galaxy.ansible.com/collivier/xtesting) leverages the
+common test case execution proposed by Xtesting. Thanks to a simple test case
+list, this tool deploys plug-and-play
+[CI/CD toolchains in a few commands](https://wiki.opnfv.org/pages/viewpage.action?pageId=32015004).
+In addition, it supports multiple components such as Jenkins and Gitlab CI
+(test schedulers) and
+[multiple deployment models](https://lists.opnfv.org/g/opnfv-tsc/message/5702)
+such as all-in-one or centralized services.
+
+[Xtesting](https://xtesting.readthedocs.io/en/latest/) and
+[Xtesting CI](https://galaxy.ansible.com/collivier/xtesting) combined meet the
+CNTT requirements about verification, validation, compliance and conformance:
```

Comment:
```suggestion
CNTT requirements about verification, validation, compliance, and conformance:
```

Reply:
it doesn't seem right in my native language. Can someone second this proposal?